packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -742,19 +742,35 @@ export const audiusBackend = ({
       const account = audiusLibs.Account.getCurrentUser()
       if (!account) return null
 
-      try {
-        const body = await getSocialHandles(account.handle)
-        account.twitter_handle = body.twitterHandle || null
-        account.instagram_handle = body.instagramHandle || null
-        account.tiktok_handle = body.tikTokHandle || null
-        account.website = body.website || null
-        account.donation = body.donation || null
-        account.twitterVerified = body.twitterVerified || false
-        account.instagramVerified = body.instagramVerified || false
-        account.tikTokVerified = body.tikTokVerified || false
-      } catch (e) {
-        console.error(e)
+      // This is saying that if there is any social info coming back from DN,
+      // then we should not attempt to fetch socials from identity.
+      // It is possible that identity has more up-to-date socials than DN,
+      // but we can live with that until we do the final social backfill from identity to DN.
+      const hasSocialsFromDn =
+        account.twitter_handle ||
+        account.instagram_handle ||
+        account.tiktok_handle ||
+        account.website ||
+        account.donation ||
+        account.verified_with_twitter ||
```

Comment:
Just double-checking the logic here. If the API returns a _defined_ / non-null value, then we trust it, right?
i.e. if it's `false`, then the DN populated it?
Just thinking that in the case where a user hasn't set any handles/website/donation, but the DN populates the fields it knows about, we would get to this line and consider a value of `false` to be not populated.

Reply:
yeah true, but i think we can live with that. the `false` value is hard to tell if its legit or not. i understand this means that we will go to identity more often than not, for now.